### DISCUSSION TITLE
Update SO from v14 to 2.x

### DISCUSSION THREAD

**Discussion by cyb3rz3us** [Security-Onion-Solutions/securityonion#5440] [General] [answered]
Hello SO Community - Based on my reading of the SO docs, it appears that upgrading from v14 to v2.x is really not an option.  Am I correct here?

**Comment by tomjones1977** [comment, marked as answer]
It is recommended that a fresh install is carried out.  I would say that given the risks of upgrade errors, and how significantly SO has changed it is worth the effort to do a fresh install from scratch.  What we did was isolate the existing storage nodes and master so that we could refer back if needed, and start a fresh install.  Once a reasonable time had passed we decommissioned the old nodes entirely.

**Reply by cyb3rz3us** [reply]
That's what I figured would be the case - thank you for confirming...
